website_sale_suggest_create_account/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Suggest to create user account when buying",
+    "summary": "Suggest users to create an account when buying in the website",
+    "version": "10.0.1.0.0",
+    "category": "Website",
+    "website": "https://www.tecnativa.com",
+    "author": "Tecnativa, "
+              "LasLabs, "
+              "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
```

Comment:
I'm good for LGPL, thanks for checking.

I'll go ahead and provide pre-approval to update any LasLabs licensing from AGPL-3 to LGPL-3 if you happen to run across the scenario again 😉